zerver/webhooks/jira/doc.md

```diff
@@ -14,8 +14,12 @@ Server versions 5.2 or greater. For older versions, you'll need our
 
 1. {!generate-webhook-url-basic.md!}
 
-1. Go to your Jira **Site administration** page, and click on **Jira**.
-   In the left sidebar, scroll down, and under **Advanced**, click
+1. Go to your Jira **Site administration** page. Click on the
+   **dropdown**( <i class="fa fa-ellipsis-h" aria-hidden="true"></i> )
+   under the **Actions** column on your **Jira** app, and select
+   the **Jira Settings** option.
```

Comment:
I think you could simplify this a bit: "under **Actions** for your **Jira** app, and select **Jira settings**."

From your screenshot of the UI here, it looks like it's "Jira settings" and not "Jira Settings" re:capitalization. Also, should we use "app" here or "product"? I ask because in the screenshot that column is labeled "product".